frontend/src/scenes/onboarding/sdks/shared-snippets.tsx

```diff
@@ -6,3 +9,23 @@ export function SessionReplayFinalSteps(): JSX.Element {
         </>
     )
 }
+
+export function PersonModeEventPropertyInstructions(): JSX.Element {
+    return (
+        <>
+            <h4>Optional: Specify person profile processing</h4>
+            <p>
+                By default, for backwards compatibility reasons, events are sent with{' '}
+                <Link to="https://posthog.com/docs/data/persons" target="_blank" targetBlankIcon>
+                    person profile processing
+                </Link>{' '}
+                enabled. This means a person profile will be created for each user who triggers an event.
+            </p>
+            <p>
+                If you want to disable person profile processing for certain events, send the event with the following
+                property:
+            </p>
+            <CodeSnippet>"$process_person_profile": false</CodeSnippet>.
+        </>
```

Comment:
Since these are new users, its best to hand hold them here and tell them how to set properties. Otherwise they need to go to the docs and look it up, which adds extra confusion and delays their onboarding.

Not sure the code below is correct, but just to illustrate an example:

i.e. something like this: 
```suggestion
                If you want to disable person profile processing for certain events, send the event with the `$process_person_profile` set to false:
            </p>
            <CodeSnippet>
            client.Enqueue(posthog.Capture{
    DistinctId: "distinct_id_of_the_user",
    Event:      "user_signed_up",
    Properties: posthog.NewProperties().
      Set("$process_person_profile", false),
  })
            </CodeSnippet>.
        </>
```

Reply:
I was being lazy here not wanting to make a custom code snippet for every library... but I guess I'll do it :)

Reply:
I added this as a follow-up instead of handling it in this PR 👍